jquery-src/jquery.calendars.chinese.js

```diff
@@ -273,7 +273,7 @@
             var monthDaysTable = LUNAR_MONTH_DAYS[year - LUNAR_MONTH_DAYS[0]];
             var intercalaryMonth = monthDaysTable >> 13;
 
-            return (intercalaryMonth && intercalaryMonth === monthIndex);
+            return (intercalaryMonth !== 0 && intercalaryMonth === monthIndex);
```

Comment:
Strictly speaking is a bug fix because `0 || something` evaluates to `0` instead of `false`, and I've documented this function to return a boolean.